lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -288,10 +290,14 @@ function cancelTerms(numerator, denominator) {
     return new CancelOutStatus(numerator, denominator, true);
   }
 
-  // case 4: the numerator is a constant and denominator is a polynomial term
+  // case 4: the numerator is a constant and denominator is a polynomial term that has a coefficient
+  // or is multiplication node
   // e.g. 2 / 4x -> 1 / 2x
+  // e.g. ignore cases like:  2 / a and 2 / x^2
```

Comment:
Oh sorry about the Sentry link. I was unaware that you can't view it! I added an extra check to ignore cases where the denominator isn't a coefficient * polynomialTerm. Without that, we would be stuck in that `if` statement.